packages/storybook-addon-designs/package.json

```diff
@@ -35,6 +35,13 @@
     "react": "^16.8.4",
     "typescript": "^3.7.0"
   },
+  "peerDependencies": {
+    "@storybook/addon-docs": "6.3.2",
+    "@storybook/addons": "6.3.2",
+    "@storybook/api": "6.3.2",
+    "@storybook/components": "6.3.2",
+    "@storybook/theming": "6.3.2"
```

Comment:
With this range specifier, every user will be forced to install `v6.3.2`. Should be something like `6.x.x` (This addon currently supports Storybook >= v6.0.0.)

Reply:
Yes, thanks!